examples/step-60/step-60.cc

```diff
@@ -364,9 +364,14 @@ namespace Step60
 
     std::unique_ptr<Mapping<dim, spacedim>> embedded_mapping;
 
-    // We do the same thing to specify the value of the function $g$,
-    // which is what we want our solution to be in the embedded space.
+    // We do the same thing to specify the value of the forcing term $f$.
     // In this case the Function is a scalar one.
+    ParameterAcceptorProxy<Functions::ParsedFunction<spacedim>>
+      embedding_value_function;
```

Comment:
embedding_dirichlet_boundary_conditions

Reply:
That's related to the value of the forcing term f, the first component of the right-hand side, which is named `embedding_rhs`, not to the bcs.

Reply:
If I understand this correctly, this is the rhs of the Poisson problem, right? I would not name this in the same way as the other term, simply because they have a different meaning. Maybe I would rename this `embedding_rhs_function`, and similarly for the parameter files.

Reply:
Right, it's the rhs for Poisson. I'll follow your suggestion.